### PR TITLE
Added Screenly section

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated page as per [copy doc](https://docs.google.com/document/d/1mXnYnpaCJznDLtYn9Q74aVeEI3iFxVOMzF0pdL7wdu0/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11058.demos.haus/core/stories
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against copy doc and see that all changes are there
- Test link and see that it works as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4724

## Screenshots 
Screenshot 2021-12-15 at 14 01 26

### CODE REVIEW

**Comment by anthonydillon** [comment]
@mtruj013 the QA steps here could be better.

**Comment by mtruj013** [comment]
> @mtruj013 the QA steps here could be better.

Yup, I noticed one more thing I had to fix after I made the PR, meant to tackle that first and then come back and finish the QA steps. They're updated now :)

File: templates/core/stories.html [hunk 1]
```diff
@@ -240,6 +264,10 @@ <h4>Cinergy makes significant digital signage saving with Screenly Pro and Ubunt
   </div>
 </section>
 
+<div class="u-fixed-width">
+  <hr class="p-separator" />
+</div>
```
Comment (anthonydillon) [inline templates/core/stories.html:269]: The spacing would look better here if this was within the same strip. I would wrap all of this together like the separator below

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Code looks good but the new separator is outside of a strip adding to much padding.

**Review by anthonydillon — APPROVED** [review]

**Pull request merged**